### ISSUE TITLE
raml doc not working

### ISSUE THREAD

**Issue by despiegk** [Jumpscale/go-raml#105]
with quite some trial and error found following url to do something

![image](https://cloud.githubusercontent.com/assets/6021844/15790725/f7af83bc-29e3-11e6-8690-eac1d645a06b.png)

please make sure that home page gets to apidocs

and that it works again

**Comment by despiegk** [comment]
it keeps on loading, doesn't come

**Comment by iwanbk** [comment]
@despiegk can you provide the .raml file?

I met this issue before, the issue was between these:
- invalid .raml
- the syntax is not supported by https://github.com/mulesoft/api-console

**Comment by despiegk** [comment]
its your example for python

On Sat, Jun 4, 2016 at 12:30 PM, Iwan Budi Kusnanto <
notifications@github.com> wrote:

> @despiegk <https://github.com/despiegk> can you provide the .raml file?
>
> I met this issue before, the issue was between these:
>
>    - invalid .raml
>    - the syntax is not supported by
>    https://github.com/mulesoft/api-console
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Jumpscale/go-raml/issues/105#issuecomment-223744036>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFvi1DJvfgaljTZlGgGAKScXvLsZL1BFks5qITeagaJpZM4It16j>
> .
>



-- 
------------------------
Kristof De Spiegeleer
+971525609014
+201206927877
+32 475405474
skype: despiegk

**Comment by iwanbk** [comment]
@despiegk 
I have fixed the python example in raml1.0-rc2  branch.

I also have created default API home page, can be accessed at http://localhost:5000 for python.
There is link to apidocs from that page, 

![screen shot 2016-06-06 at 1 28 39 pm](https://cloud.githubusercontent.com/assets/977552/15813669/1381715a-2beb-11e6-9fdb-1580146813fa.png)

If we click `API Docs` link, we will go to api docs page

![screen shot 2016-06-06 at 1 28 47 pm](https://cloud.githubusercontent.com/assets/977552/15813701/41b95880-2beb-11e6-9f9f-730accba9070.png)

**Comment by iwanbk** [comment]
```
with quite some trial and error found following url to do something
```

I have improved the docs https://github.com/Jumpscale/go-raml/tree/raml1.0-rc2#simple-home-page-and-api-docs

```
Both servers have simple home page that can be accessed in http://localhost:8000 for Go and http://localhost:5000 for Python.

There is API Docs link that will go to auto generated API documentation powered by api-console
```

**Comment by despiegk** [comment]
Please merge to master

**Comment by iwanbk** [comment]
merged in #106
